docs/tables/aws_clb_access_log/index.md

```diff
@@ -57,7 +57,9 @@ select
   backend_ip,
   elb_status_code,
   backend_status_code,
-  request
+  request_http_version,
+  request_http_method,
+  request_url
```

Comment:
I propose we use same name for these fields as we approved for nginx/apache: 
- `request_uri`
- `request_method`
- `server_protocol`

@cbruno10 - agree?